metadata.json

```diff
@@ -62,6 +62,14 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">=4.16.0 <5.0.0"
+    },
+    {
+      "name": "puppetlabs/dsc",
+      "version_requirement": ">=1.4.0 <2.0.0"
+    },
+    {
+      "name": "puppetlabs/acl",
```

Comment:
Even if required by this commit, I'd suggest to not add this as an hard dependency in metadata.json.
The (great) dsc module provides an hell of plugins that are synced to every client (both linux, where they are not used, and windows). I'd suggest to document the requirement for Windows nodes when the windows_service_user param is set. my2c

Reply:
+1